src/plugins/audio-stream/backend.ts

```
@@ -0,0 +1,137 @@
+}
+
+export const backend = createBackend<
+  {
+    app: Hono;
+    server?: ServerType;
+  },
+  AudioStreamConfig
+>({
+  app: new Hono()
+    .get('/stream', (ctx) => {
+      // Per-song TEXT metadata is carried in-band via chained Ogg logical streams
```

ESLint

[error] Delete `··`

(prettier/prettier)